packages/preview/clean-math-thesis/0.4.0/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024-present Sebastian Eberle
```

Comment:
Could you update the date here?

Reply:
The project started in 2024 and is going until the present. So shouldn't this be fine?

Reply:
No worries :)
Thanks for merging!